src/Faulkner/state_manager.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, goal_pos):
         self.err_pos = 0.0
 
         # action state
-        self.action_state = 0
+        self.action_state = 1 # start by fixing yaw
```

Comment:
Why?

Reply:
because we need to be at the correct angle first to determine if any objects in the environment obstruct our path; conversely, starting off with avoiding obstacles may cause the robot to follow walls that it doesn't need to and would confuse the user